client/components/environment-badge/style.scss

```diff
@@ -59,8 +59,8 @@
 			color: var( --color-warning-80 );
 		}
 		&.is-wpcalypso {
-			background-color: var( --color-green-5 );
-			color: var( --color-green-80 );
+			background-color: var( --studio-celadon-40 );
```

Comment:
Open to suggestions on a color here 🙂

Reply:
```css
background: var( --color-success-50 );
color: var( --color-text-inverted ); /* white */
```

Unless we’re using green for any other environment, I’d go with the above for more consistency. 🙂

I like celadon too, but we’d be mixing semantic and non-semantic properties in that file (`color-error` vs. `studio-red`). And `*-50` properties are automatically always AA-accessible on white.

If we decided to ditch the semantics-driven colors for the badges, we have some nice colors that we barely use in Calypso: `purple`, `celadon`, `orange`. We could reassign them and make the badges stand out from the rest of the UI a little more.

Reply:
The semantic mappings here are a bit odd. Right now it appears to be:
dev -> error
wpcalypso -> success
staging -> warning
horizon/feedback -> primary-light
branch -> neutral-70

These will change with admin theme changes, so it's not always going to be obvious which env you're on by color. I think we should move to studio colors so it's always the same, no matter which color theme is chosen.

Reply:
> I think we should move to studio colors so it's always the same, no matter which color theme is chosen.

Agreed, let's move to a fixed, unique color for each environment. @porada happy to use whichever colors you suggest 😉

Reply:
I don’t think the color itself should communicate the environment, but I’d like our color assignment to make some sense. I can definitely help with that.

> I think we should move to studio colors so it's always the same, no matter which color theme is chosen.

👍

Just so that I have more understanding in what those stages mean, let me know if the lingo here is correct:

* `dev`: work in progress
* `branch`: also work in progress, but can be previewed with `calypso.live`
* `staging`: self-explanatory
* `horizon`: a post-staging but pre-production build served to a fraction of our audience
* `wpcalypso`: production, served to 100% of our audience

Reply:
* `dev`: local development build running at `http://calypso.localhost:3000`
* `branch` and `wpcalypso`: Live branches can be viewed at [https://calypso.live/](https://calypso.live/?branch=update/wpcalypso-badge-color), they use the same environment setup and will have same badge as https://wpcalypso.wordpress.com - which is a sort of public alpha release.
* `staging`: self-explanatory -  private pre-production builds and separate environment at https://wordpress.com
* `horizon`: public beta/testing/feedback environment at https://horizon.wordpress.com

`horizon` and `wpcalypso` have their own favicons, it would be nice to align with them and/or update them to match.

Reply:
How about these combos then?

| Environment | Background | Text |
| - | - | - |
| `dev` | `--studio-purple-20` | `--studio-purple-90` |
| `branch` | `--studio-celadon-20` | `--studio-celadon-90` | 
| `wpcalypso` | `--studio-green-20` | `--studio-green-90` |
| `staging` | `--studio-orange-20` | `--studio-orange-90` |
| `horizon` | `--studio-blue-20` | `--studio-blue-90` |

Reply:
I've updated the applied those suggestions and updated the description with screenshots. What do you think?